localVariables.js

```diff
@@ -0,0 +1,4 @@
+const secretKey = "0a4cf289b12ab2bf42e5e112d07b7a6df2a7753e61297e11d926550e27e55a45"
+const apiKey = "3f55ff40e6c2ee7ba66cee52546a8d6da9456f88a2719ee9d54e1329aa03db57"
```

Comment:
Put this into a .env file and never share it!

Reply:
Completed
Updated .env package handler. 
deleted keys from files. 
updated files to work with .env handler.